examples/authsignal-passwordless/pages/api/login.ts

```diff
@@ -0,0 +1,15 @@
+import { Authsignal } from "@authsignal/node";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const authsignal = new Authsignal({ secret: process.env.AUTHSIGNAL_SECRET! });
+
+export default async function login(req: NextApiRequest, res: NextApiResponse) {
+  const { email } = req.body;
```

Comment:
could we maybe make it more explicit that this endpoint accepts POST? E.g. return 400 if it's a different method, same on logout.

Should there be some kind of CSRF protection added?

Reply:
I've added the method checks to login/logout. Re. CSRF, I left the routes as same-origin by default, otherwise I could potentially add CSRF tokens with something like [next-csrf][https://github.com/j0lv3r4/next-csrf] but it might overcomplicate the example somewhat? (I also added a few comments to explain how authentication in `/api/finalize-login` works.)